imap_data_access/cli.py

```diff
@@ -80,6 +80,11 @@ def _print_query_results_table(query_results: list[dict]):
     print(hyphens)
 
 
+def extract_version_number(str_version: str) -> int:
```

Comment:
```suggestion
def _extract_version_number(str_version: str) -> int:
```
Pretty nitpicky here but I think this function could be considered 'internal only' to this module, so by [PEP8 conventions](https://peps.python.org/pep-0008/#descriptive-naming-styles) should be named with a leading underscore